app/controllers/tasklist_controller.rb

```diff
@@ -0,0 +1,5 @@
+class TasklistController < ApplicationController
+  def show
+    @tasklist = TasklistContent.learn_to_drive_config
```

Comment:
We should follow the convention implemented in other controllers, of not using instance variables

```
      render :show, locals: {
        tasklist: TasklistContent.learn_to_drive_config
      }
```

Reply:
I've changed it to match, thanks